R/estimate_uncertainty_retro.R

```diff
@@ -74,6 +75,7 @@ estimate_uncertainty_retro <- function(
     max_delay = ncol(reporting_triangle) - 1,
     structure = detect_structure(reporting_triangle),
     delay_pmf = NULL,
+    uncertainty_model = fit_by_horizon,
```

Comment:
As in the main comment I can't see why you would need this? Have you tested it without? leaving everything else as is?